packages/extension-ui/src/components/RadioGroup.tsx

```diff
@@ -28,14 +36,55 @@ const RadioGroup: React.FC<Props> = ({ className, defaultSelectedValue, onSelect
     [onSelectionChange]
   );
 
+  console.log('options', options);
+
+  const { alephOptions, otherOptions } = options.reduce(
+    (acc: { alephOptions: Option[]; otherOptions: Option[] }, option) => {
+      const isAleph = withTestNetwork ? option.text.includes('Aleph Zero') : option.text === 'Aleph Zero';
+
+      if (isAleph) {
+        acc.alephOptions.push(option);
+      } else if (withTestNetwork || !option.text.includes('Aleph Zero')) {
+        acc.otherOptions.push(option);
+      }
+
+      return acc;
```

Comment:
I saw this in another comment already, so just a reminder that mutations should not be used in _functional_ methods like `reduce()`, so:
```suggestion
      if (isAleph) {
        return {
          ...acc,
          alephOptions: [...acc.alephOptions, option],
        }
      }
      
      if (withTestNetwork || !option.text.includes('Aleph Zero')) {
        return {
          ...acc,
          otherOptions: [...acc.otherOptions, option],
        }
      }

      return acc;
```
is more _proper_ in spite of being longer. While this seems suboptimal in terms of readability at first, it actually makes it more readable due to no mutations and especially pays off during refactors / debugging.